pkg/crcontrollers/drive/drivecontroller.go

```diff
@@ -118,12 +125,15 @@ func (c *Controller) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		// TODO need to check CSI status here since volume might not be removed
 		volume := c.crHelper.GetVolumeByLocation(id)
 		if volume == nil || volume.Spec.CSIStatus == apiV1.Removed {
-			// TODO need to call drive manager to start LED locate
 			drive.Spec.Usage = apiV1.DriveUsageRemoved
+			status, err := c.driveMgrClient.Locate(ctx, &api.DriveLocateRequest{Action: apiV1.LocateStart, DriveSerialNumber: drive.Spec.SerialNumber})
```

Comment:
Please add comments that we need to send alert here when LED locate failed

Reply:
Done